pennylane/tape/measure.py

```diff
@@ -78,6 +78,16 @@ def __init__(self, return_type, obs=None, wires=None, eigvals=None):
         # Queue the measurement process
         self.queue()
 
+    def __repr__(self):
+        """Representation of this class."""
+        if self.obs is None:
+            return "{}(None)".format(self.return_type.value)
```

Comment:
This line is needed because some tests construct Processes with None observables.

Reply:
If `self.obs` is none, should we instead display the wires?
```suggestion
            return "{}(wires={})".format(self.return_type.value, self.wires)
```

Although, `qml.state` does not specify wires, so it will just be blank in that case.